x/oracle/types.go

```diff
@@ -15,33 +18,49 @@ type BridgeClaim struct {
 	Nonce          int            `json:"nonce"`
 	EthereumSender string         `json:"ethereum_sender"`
 	CosmosReceiver sdk.AccAddress `json:"cosmos_receiver"`
+	Validator      sdk.AccAddress `json:"validator"`
 	Amount         sdk.Coins      `json:"amount"`
 }
 
 // NewBridgeClaim returns a new BridgeClaim with the given data contained
-func NewBridgeClaim(nonce int, ethereumSender string, cosmosReceiver sdk.AccAddress, amount sdk.Coins) BridgeClaim {
+func NewBridgeClaim(nonce int, ethereumSender string, cosmosReceiver sdk.AccAddress, validator sdk.AccAddress, amount sdk.Coins) BridgeClaim {
 	return BridgeClaim{
 		Nonce:          nonce,
 		EthereumSender: ethereumSender,
 		CosmosReceiver: cosmosReceiver,
+		Validator:      validator,
 		Amount:         amount,
 	}
 }
 
 // BridgeProphecy is a struct that contains all the metadata of an oracle ritual
 type BridgeProphecy struct {
-	Status        string        `json:"status"`
 	Nonce         int           `json:"nonce"`
+	Status        string        `json:"status"`
 	MinimumClaims int           `json:"minimum_claims"` //The minimum number of claims needed before completion logic is checked
```

Comment:
Updating `MinimumClaims` to `MinimumPower` in the `BridgeProphecy` struct makes sense, as many parts of the application interact with the data structure.